ospd/ospd.py

```diff
@@ -1181,7 +1194,8 @@ def run(self, server: BaseServer) -> None:
         """ Starts the Daemon, handling commands until interrupted.
         """
 
-        server.start(self.handle_client_stream)
+        while not self.initialized:
+            time.sleep(5)
```

Comment:
Not sure about this waiting procedure. Could it be ever possible that init is run asynchronously and doesn't return before run is called? I can't image this currently.

Reply:
You are right. run() will be always called after init() finished. So, I removed this while.